README.md

```diff
@@ -39,33 +41,29 @@ Quickwit is a cloud-native search engine for log management & analytics. It is d
 
 # 💡 Features
 
-- Index data persisted on object storage (AWS S3, GCS, MinIO, Ceph)
-- Ingest JSON documents with or without a strict schema
+- Cloud-scale: K8s-native, decoupled compute & storage
+- Sub-second full-text search on object storage (AWS S3, Azure...)
+- Sleep like a log: all your indexed data is safely stored on object storage
 - Ingest & Aggregation API Elasticsearch compatible
-- Lightweight Embedded UI
-- Runs on a fraction of the resources: written in Rust, powered by the mighty tantivy
-- Works out of the box with sensible defaults
-- Optimized for multi-tenancy. Add and scale tenants with no overhead costs
-- Distributed search
-- Cloud-native: Kubernetes ready
+- Search stream API for full-text search in ClickHouse
+- Multi-tenancy
 - Add and remove nodes in seconds
-- Decoupled compute & storage
-- Sleep like a log: all your indexed data is safely stored on object storage
-- Ingest your documents with exactly-once semantics
-- Kafka-native ingestion
-- Search stream API that notably unlocks full-text search in ClickHouse
+- Distributed search
+- Distributed indexing with Kafka
+- Native support for Kafka & Kinesis with exactly-once semantics
+- Schemaless or strict schema indexing
+- Works out of the box with sensible defaults
+- Lightweight Embedded UI
 
 ### 🔮 Roadmap
-- [Quickwit 0.5 - Q2 2023](https://github.com/quickwit-oss/quickwit/projects/6)
-  - Distributed and replicated ingestion queue
-  - Tiered storage (local drive, block storage, object storage)
-  - Native support for OpenTelemetry exporters (logs and traces)
-  - Jaeger integration
+- [Quickwit 0.5 - Q1 2023](https://github.com/quickwit-oss/quickwit/projects/6)
   - Grafana data source
+  - Native support for OpenTelemetry
+  - REST API for managing indexes
 - [Long-term roadmap](ROADMAP.md)
+  - Distributed indexing
```

Comment:
Maybe this needs clarifying as just above we mention we support distributed indexing with kafka.

Reply:
All good now, the new roadmap doesn't have this part